### PR TITLE
Upgrade package dependencies

### PR DESCRIPTION
**Pull request by AhnpGit**
ts-jest just released their new package version which removes `mapCoverage` to be in sync with Jest. I think it's time for us to be in sync with them too.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,9 @@
 ## Changelog (master)
+* Chore: Upgrade ts-jest and remove `mapCoverage` from `jest-preset` ([#109](https://github.com/thymikee/jest-preset-angular/pull/117))
 
 ### v5.1.0
 * Feature: Simplify installation by adding @types/jest as a package dependency ([#116](https://github.com/thymikee/jest-preset-angular/pull/116))
-* Feature: Move serializers setup to jest config to be possible to override them ([#126](https://github.com/thymikee/jest-preset-angular/pull/116))
+* Feature: Move serializers setup to jest config to be possible to override them ([#117](https://github.com/thymikee/jest-preset-angular/pull/116))
```
Comment (thymikee) [inline CHANGELOG.md]: Can you revert this change?

Comment (AhnpGit) [reply CHANGELOG.md]: Ok
